Add v1.x.x Q&A to FAQ (#238)
Azure/ALZ-Bicep: docs/wiki/FAQ.md

## Questions & Answers

## Why isn't `ALZ-Bicep` v1.x.x yet?

Great question! Put simply this is because Bicep itself is not on a v1.X.X release yet which means there is a possibility for larger breaking changes still to occur in Bicep itself (upstream) which will in turn mean that these changes will have to be integrated into `ALZ-Bicep`.

This however, does not mean you shouldn't use ALZ-Bicep or Bicep as they are fully supported and ready for production usage.

> As we use GitHub releases and tags you can clone and pin to a specific version in your codebase and upgrade when you are ready too and if you want to 👍

Reviewer: ```suggestion
As we use GitHub releases and tags you can clone and pin to a specific version in your codebase and upgrade when you are ready and willing to do so 👍
```
